sftp/2.5/modules/ROOT/pages/sftp-documentation.adoc

```diff
@@ -408,11 +408,35 @@ NOTE: An additional check is performed if the file already exists, thus the FTP
 * OVERWRITE
 * APPEND
 * CUSTOM_APPEND
-* CREATE_NEW |  A `FileWriteMode`. Defaults to `OVERWRITE` |  `OVERWRITE` |
+* CREATE_NEW |  A `FileWriteMode`.
+
+* OVERWRITE
++
+If the file to be written already exists, then it should be overwritten.
+* APPEND
++
+If the file to be written already exists, then the content should be appended to that file.
+* CUSTOM_APPEND
++
+If the file to be written using the `CUSTOM` write strategy already exists, then the content should be appended to that file.
+* CREATE_NEW 
++
+A new file should be created and an error should be raised if the file already exists.
```

Comment:
Create a file and raise an error if the file exists. (To avoid "should" and "already". "new" is considered extraneous.)